### DISCUSSION TITLE
How to set all the other field value with first one

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [react-hook-form/react-hook-form#9182] [Q&A]
I have a dynamic field where if user select 3 or more product there were open new field according to user chosen number 
Now i want if he fill the first one in want rest of the field have same value of the first one after that if user want to change any field can update but default values should be first one either he change or not rest of field have same default values
My codesandbox link : https://codesandbox.io/s/react-hook-form-forked-to7c34
![image](https://user-images.githubusercontent.com/98738905/194825858-4f9e546d-e572-439f-9163-d7f0f156ff67.png)

**Comment by leapful** [comment]
@abdulmateentechbits 

You can use native input attribute "defaultValue" to apply value of first product to all products.

Working demo: https://codesandbox.io/s/react-hook-form-forked-75xg2f?file=/src/components/ui/inputs/InputField.jsx

**Reply by abdulmateentechbits** [reply]
@leapful  thanks for your reply 
Now the issue were when I submit it the default values were empty

**Reply by leapful** [reply]
@abdulmateentechbits 

In this case, you can simplify it with "replace" from "useFieldArray".

Working demo: https://codesandbox.io/s/react-hook-form-forked-w7zxp2?file=/src/components/forms/NestedInput.jsx
